lib/bitcoin/BitcoinClient.ts

```diff
@@ -42,12 +41,21 @@ export default class BitcoinClient {
     private bitcoinPeerUri: string,
     bitcoinRpcUsername: string | undefined,
     bitcoinRpcPassword: string | undefined,
-    bitcoinWalletImportString: string,
+    bitcoinWalletImportString: string | undefined,
+    bitcoinWallet: IBitcoinWallet | undefined,

```

Comment:
What about:
```suggestion
    bitcoinWalletOrImportString: IBitcoinWallet | string,
```

Reply:
Oh my!! I love javascript :) Let me try and will update.